### DISCUSSION TITLE
Error on Enabling kong api gateway prometheus plugin in Docker

### DISCUSSION THREAD

**Discussion by Madurad** [Kong/kong#10780] [Help] [answered]
Hello,

I configured prometheus plugin in my kong declarative config file using the below configuration, but getting the error when I run it on kong docker image.

Error : 

```
`Error: 
/usr/local/share/lua/5.1/kong/cmd/config.lua:110: Failed parsing:
in 'plugins':
  - in entry 4 of 'plugins':
    in 'config':
      in 'latency_metrics': unknown field
      in 'bandwidth_metrics': unknown field
      in 'status_code_metrics': unknown field
stack traceback:
        [C]: in function 'error'
        /usr/local/share/lua/5.1/kong/cmd/config.lua:110: in function 'cmd_exec'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:88: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:88>
        [C]: in function 'xpcall'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:88: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:45>
        /usr/local/bin/kong:9: in function 'file_gen'
        init_worker_by_lua:49: in function <init_worker_by_lua:47>
        [C]: in function 'xpcall'
        init_worker_by_lua:56: in function <init_worker_by_lua:54>`

```
Configuration on declarative yaml

`- name: prometheus
    config:
      bandwidth_metrics: true
      latency_metrics: true
      status_code_metrics: true`

Docker image : kong:2.1.1

Greatly appreciate if anyone can give me a help.

Thanks.

**Comment by samugi** [comment, marked as answer]
Hi @Madurad , are you using Kong version 2.1.1 as mentioned in your comment? If so, this is expected, the parameters you are attempting to configure where only included after version 3.0. I would recommend trying this again in a more recent (or the latest available) version of Kong.

**Reply by Madurad** [reply]
Hi @samugi thanks for the reply, and I tried with v3.1.1 with my docker image and that got worked..Thanks for your great guidance..
